%Analysis_methods.tex
\documentclass [12pt, a4paper, twoside]  {article}
\usepackage [a4paper, left=30mm, right=30mm, top=20mm, bottom=20mm] {geometry}

%%% Paragraph formatting
\setlength{\parindent}{0pt} %new paragraph indent is 0
\setlength{\parskip}{1em}	%new paragraphs are indicated by a new line
\renewcommand{\baselinestretch}{1.2}

%%% Making sure line numbers are included
\RequirePackage{lineno}
\usepackage{lineno}

%%% Arranging graphics parameters
\usepackage [utf8] {inputenc}
% use this to put captions to the right of figures
\usepackage{sidecap}
\sidecaptionvpos{table}{c}
  
% allows exact specification of where figures are placed in the main text
\RequirePackage{float}
\usepackage{float}

\RequirePackage{ graphicx}
\usepackage { graphicx }
\graphicspath{ {./figs/} } %## Commented out because the path doesn't work (7/10/17)
\usepackage[labelfont=bf]{caption}
\DeclareCaptionLabelFormat{bf-parens}{\textbf{#1#2}}\captionsetup{labelsep=quad}
\usepackage{subcaption}
\usepackage{gensymb}
\usepackage{enumitem}
\setlist{nosep}

%%% Bibliography formatting
%% Remember that, in order to get in-text citations and the correct bibliography in the final PDF, you need to do a LaTeX (or XeLaTeX) run, then a BibTex run, followed by two more (Xe)LaTex runs. This should then solve any issues with in-text citations appearing as (?)
\usepackage{filecontents}
\usepackage[round]{natbib}	% See https://gking.harvard.edu/files/	natnotes2.pdf for more information

\usepackage{array}

\usepackage{multicol}

%%% Ensures that the text will be crossreffed with hyperlinks
\usepackage{hyperref}
	\hypersetup{
		colorlinks=true
		linkcolor=blue
		filecolor=magenta
		urlcolor=true}

%% My own commands to make things a bit quicker to type
\newcommand{\super}{\textsuperscript}
\newcommand{\sub}{\textsubscript}

\begin{document}

All statistical analysis was carried out in R [citation]. Data and analyses are available at this paper's Github page.

\section*{Sequence analysis}
Twelve clones from each replicate at the three timepoints considered had their two CRISPR loci analysed using PCR to determine spacer acquisition. Clones identified as carrying spacers then had the locus of interest sent sent for Sanger sequencing (Source Bioscience, UK). CRISPR1 and CRISPR3 were sequenced using primers [CR1 primer sequence] and [CR3 primer sequence]. The entire sequenced locus was mapped against the $\phi 2972$ genome (Accession: NC\textunderscore 007019.1) using BLAST. Putative spacer sequences were hits approximately 30bp. Genomic location and read direction of each protospacer hit were parsed using Geneious v9.1.8 \citep{kearse2012geneious}. False positive differences in hit location between putative spacer sequences generated by the BLAST algorithm were identified and the data cleaned accordingly. 

\section*{Phage survival}
Phage survival was analysed using the \texttt{survival} package \citep{survival}. A Cox proportional hazards model was used to assess the effect of initial phage titer on phage survival over the course of the experiment. Hazard ratio coefficients express the relative risk of phage extinction over time. 

\section*{Inferring coevolutionary dynamics}
Stamp assays were used to assess infectivity and resistance of phage and bacterial phenotypes, respectively. Infectivity was measured as the proportion of hosts infected, with the inverse for host resistance. 


\clearpage
%% Set up the bibliography	
\bibliographystyle{./humannatemph.bst}				% default "author-year" style for natbib
\bibliography{../../../references.bib}

\end{document}
